recipes/trisbm/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "nsbm" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fvalle1/trisbm/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 268897a510cd0de903f8069376e4124544c515b1c070af6af31fea0b0ef7628c
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
```

If (and only if!) this can be `noarch: python`:

```suggestion
    - python >=3.6
```